discovery_engine_core/providers/src/lib.rs

```diff
@@ -33,4 +33,4 @@ mod newscatcher;
 
 pub use client::{Client, HeadlinesQuery, NewsQuery};
 pub use filter::{Filter, Market};
-pub use newscatcher::{Article, Topic};
+pub use newscatcher::{Article, Response as NewscatcherResponse, Topic};
```

Comment:
can `Response` be confused with anything else that the crate exports? otherwise we don't have to rename it. if we have to rename it, we should do so consistently also for the other exported types of `newscatcher::`.